models/inventory.js

```diff
@@ -0,0 +1,48 @@
+module.exports = function(sequelize, DataTypes) {
+    const Inventory = sequelize.define("Inventory", {
+        stock: {
+            type: DataTypes.INTEGER(10),
+            allowNull: false,
+            validate: {
+                isNumeric: true
+            }
+        },
+        createdAt: {
+            allowNull: false,
+            type: DataTypes.DATE,
+            defaultValue: sequelize.literal('CURRENT_TIMESTAMP')
+        },
+        updatedAt: {
+            allowNull: false,
+            type: DataTypes.DATE,
+            defaultValue: sequelize.literal('CURRENT_TIMESTAMP')
+        }
+    },{
+        freezeTableName: true
+    });
+
+
+    return Inventory;
+}
+
+
+
+
+
+// store_id: {
```

Comment:
Comments on page can be left during development, however, be sure to leave notes indicating that it is to saved their for future use. Be sure to annotate your commented code.